core/spec.md

```diff
@@ -80,12 +80,12 @@ to a "Schema" document - all within the same Registry instance:
 
 For easy reference, the JSON serialization of a Registry adheres to this form:
 
-``` text
+```yaml
```

Comment:
I chose the YAML formatting here because it works surprisingly well with the notation you use. It's not valid JSON either. 

With YAML you'll get somewhat decent visual highlighting. But technically it's not correct. Falling back to ``` or ```text would be an alternative.

Reply:
ok if `yaml` works let's do that!